### PR TITLE
[modbus.studer] Update README.md

### PR DESCRIPTION
**Pull request by einstein99**
As i spent hours to figure out the correct document to read about the address scheme, this might help others to avoid this and get this plugin to the same "copy and paste" use  as the others, even if the existing scheme already is enough to get it working.

### CODE REVIEW

**Comment by fwolter** [comment]
@giovannimirulla Can you take a look?

@einstein99 You need to sign-off your commits. See https://www.openhab.org/docs/developer/contributing.html#sign-your-work The necessary commands are listed when you click on "Details" at the DCO check below. Alternatively, you can post the sign-off tag here.

**Comment by einstein99** [comment]
Signed-off-by: Michael Saalfrank <github@saalfranks.de>

File: bundles/org.openhab.binding.modbus.studer/README.md [hunk 1]
```diff
@@ -28,6 +28,25 @@ For defining a thing textually, you have to find out the start address of the mo
...
+| 10     | Multicast Xtender     |
+| 11-19  | Xtender 1-9           |
+| 20     | Multicast Variotrack  |
+| 21-35  | Variotrack 1-15       |
+| 40     | Multicast Variostring |
+| 41-55  | Variostring 1-15      |
+| 61     | BSP/Xcom-CAN          |
+|--------|-----------------------|
+
+More Details about that can be found in the technical specification and appendix for Studer RTU Modbus protocoll. Check default config (dip switches 1 and 2 off) while configuring the pin-out on the RS-485!
```
Comment (fwolter) [inline bundles/org.openhab.binding.modbus.studer/README.md:45]:
```suggestion
More Details about that can be found in the technical specification and appendix for Studer RTU Modbus protocol. Check default config (dip switches 1 and 2 off) while configuring the pin-out on the RS-485!
```

**Review by fwolter — APPROVED** [review]
LGTM

@giovannimirulla Please comment if you have any objections. I merge it for now.

**Comment by giovannimirulla** [comment]
Sorry @fwolter, I missed the notification. No objection. The documentation is now more detailed

**Comment by giovannimirulla** [comment]
> LGTM
> 
> 
> 
> @giovannimirulla Please comment if you have any objections. I merge it for now.

Sorry @fwolter, I missed the notification. No objection. The documentation is now more detailed

**Pull request merged**